lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -634,10 +634,10 @@ def _user_certificate_info(self, user, context, course_grade, bulk_certs):
             is_whitelisted,
             bulk_certs.certificates_by_user.get(user.id),
         )
-        TASK_LOG.info(
-            u'Student certificate eligibility: %s '
-            u'(user=%s, course_id=%s, grade_percent=%s letter_grade=%s gradecutoffs=%s, allow_certificate=%s, '
-            u'is_whitelisted=%s)',
+        TASK_LOG.debug(
```

Comment:
How can we see debug logs? There are getting logged somewhere, so I don't understand the reason for changing the type.

Reply:
Debug logs are not logged to Splunk, so we will have reduced logs on Splunk.

Reply:
Debug logs are available in Sandobx

Reply:
Is it something worth debugging in the sandbox? I just want to identify its utility and if it is not needed, we can remove it altogether. Can we ask the person who added that log in the first place if it is needed anymore?

Reply:
@DawoudSheraz I don't think we need the log.